packages/ui/src/components/Loading/Loading.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { motion } from "framer-motion";
+import { motion, useReducedMotion } from "framer-motion";
```

Comment:
This was the big one for Chromatic.  But we do have motion in use in some other components like the accordion and repeater.  Worth updating those, too?

Reply:
OH right! I forgot about those as well as the repeater. We also have some opacity based animations but those should be fine since opacity isnt motion 👍 

[add reduced motion to remaining components](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7878/commits/0b725260555412f2e3d1fc443f9a8cbaab64bf52)